src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -450,7 +497,14 @@ public Task<RecordLatestMatureDepositsResult> RecordLatestMatureDepositsAsync(IL
                                     }
                                     else
                                     {
-                                        haveSuspendedTransfers = true;
+                                        if (!res.Reject)
+                                        {
+                                            haveSuspendedTransfers = true;
+                                        }
+                                        else
+                                        {
+                                            status = CrossChainTransferStatus.Rejected;
+                                        }
```

Comment:
Can we clean this set of if/else statements up so it's easier to read?
```
if(res.Success) {
    ...
}
else if(res.Reject) {
    ...
}
else {
    ...
}
```

Reply:
Done.